developer-docs-site/docs/index.md

```diff
@@ -8,9 +8,11 @@ hide_table_of_contents: true
 
 # Aptos Developer Documentation
 
-Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
+Welcome! At Aptos Labs we are building a Layer 1 for everyone. Aptos supports the blockchain-specific [Move programming language](https://github.com/move-language/move) in its purest form, derived directly from Rust. Aptos let's you write smart contracts in Move without needing to learn complex modifications to the language itself. And the results are lightning-speed transactions created with little effort and executed in parallel.
```

Comment:
My reservations on this paragraph:
1. This has a marketing copy feel, with "lightning-speed transactions..." and also singling out Move. I believe a developer who lands on this docs page is already sold on everything Aptos or is already curious about how our dev tools look like. 
2. I am opposed to excessive anthropomorphizing software ("Aptos lets you write...")
3. I suggest we do not make this change.

Reply:
OK, reverting changes to the intro.